config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleEntityBuilder.java

```diff
@@ -52,8 +52,15 @@
             List<Entity> entities = new ArrayList<>();
             entityBuilders.forEach(builder -> entities.addAll(builder.build(bundle, bundleType, document)));
             final Element fullBundle = bundleDocumentBuilder.build(document, entities);
+            BundleMetadata fullBundleMetadata = null;
```

Comment:
is it not required to generate metadata for build-environment-bundle task ?
In annotated entity case, we are generating metadata for both the tasks

Reply:
It is not required, we should correct it for annotated entity case as well

Reply:
BTW, I do not think we are generating metadata for build-environment-bundle task (annotated case), as annotations are not ignored for env entities.